sensor_msgs/Image  image_input
---
int8 res